pkgs/development/tools/refmt/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "refmt";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "rjeczalik";
+    repo = "refmt";
+    rev = "v${version}";
+    sha256 = "sha256-HiAWSR2S+3OcIgwdQ0ltW37lcG+OHkDRDUF07rfNcJY=";
+  };
+
+  vendorSha256 = "sha256-MiYUDEF9W0VAiOX6uE8doXtGAekIrA1cfA8A2a7xd2I=";
+
+  meta = with lib; {
+    description = "Reformat HCL <-> JSON <-> YAML";
+    homepage = "https://github.com/rjeczalik/refmt";
+    license = licenses.agpl3;
```

One more thing, sorry^^

```suggestion
    license = licenses.agpl3Only;
```

---

Done!